src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/documentation.rs

```diff
@@ -0,0 +1,184 @@
+//! Documentation view visualization generating and presenting Enso Documentation under
+//! the documented node.
+
+use crate::prelude::*;
+
+use crate::graph_editor::component::visualization;
+use crate::graph_editor::component::visualization::Instance;
+use crate::graph_editor::component::visualization::Data;
+use crate::graph_editor::component::visualization::Signature;
+use crate::graph_editor::component::visualization::Path;
+use crate::graph_editor::component::visualization::Format;
+use crate::graph_editor::component::visualization::DataError;
+use crate::graph_editor::component::visualization::Definition;
+
+use enso_frp as frp;
+use ensogl::display::DomSymbol;
+use ensogl::display::scene::Scene;
+use ensogl::display;
+use ensogl::system::web;
+use ensogl::system::web::StyleSetter;
+use ast::prelude::FallibleResult;
+
+
+// =================
+// === Constants ===
+// =================
+
+pub const DOC_VIEW_WIDTH  : f32 = 300.0;
+pub const DOC_VIEW_MARGIN : f32 = 15.0;
+
+const CORNER_RADIUS   : f32  = crate::graph_editor::component::node::CORNER_RADIUS;
+/// Content in the documentation view when the data is yet to be received.
+const PLACEHOLDER_STR : &str = "<h3>Enso Documentation Viewer</h3><p>No documentation available</p>";
```

Comment:
This doesnt make sense to me. Documentation tells "Content in the documentation view when the data is yet to be received". So this displays when Im waiting for the server to send me documentation, right? If so, why I will see "No documentation available" there? Moreover, lets not repeat everywhere name `Enso`. It doesn't make sense. Just use the smallest amount of words possible not to clutter the workspace